RecoParticleFlow/PFClusterProducer/plugins/PFClusterProducer.cc

```diff
@@ -107,6 +118,14 @@ PFClusterProducer::PFClusterProducer(const edm::ParameterSet& conf)
 }
 
 void PFClusterProducer::beginRun(const edm::Run& run, const edm::EventSetup& es) {
+  if (cutsFromDB) {
+    const HcalPFCuts& hcalCuts = es.getData(hcalCutsToken_);
+    std::unique_ptr<HcalPFCuts> paramPF_;
+    paramPF_ = std::make_unique<HcalPFCuts>(hcalCuts);
```

Comment:
The `setTopo()` call was moved to the ESProducer, but this code still makes a copy? Is that necessary? Could the `hcalCuts` be used directly?

Reply:
No, es products are immutable. Have to make a copy.

Reply:
Yes, at this point the copy is no longer necessary.